src/infra/flow-system-inmem/src/services/flow/flow_service_inmem.rs

```diff
@@ -214,7 +214,7 @@ impl FlowServiceInMemory {
             None => {
                 let mut flow = self.make_new_flow(flow_key.clone(), trigger).await?;
 
-                let next_activation_time = schedule.next_activation_time(start_time);
+                let next_activation_time = schedule.next_activation_time(start_time).int_err()?;
```

Comment:
This needs more careful handling. If CRON expression may get out of next values, we should not panic, and should simply skip scheduling the next flow, as it will never happen again.